operators/skupper-operator/1.4.3/manifests/skupper-operator.v1.4.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,334 @@
+                image: quay.io/skupper/site-controller@sha256:77388f3353f103ba453f6152ea7c4b24212212a0798dc624cd928d36ba6c75ca
+                name: manager
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: skupper-site-controller
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - statefulsets
+          - daemonsets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          - pods/exec
+          - secrets
+          - configmaps
+          - serviceaccounts
+          - services
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          - networkpolicies
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - projectcontour.io
+          resources:
+          - httpproxies
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        serviceAccountName: skupper-site-controller
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - skupper
+  - service
+  - mesh
+  - van
+  links:
+  - name: Skupper Operator
+    url: https://github.com/skupperproject/skupper-operator
+  maintainers:
+  - email: skupper@googlegroups.com
+    name: Skupper Community
+  maturity: alpha
+  provider:
+    name: Skupper Project
+    url: https://skupper.io
+  replaces: skupper-operator.v1.4.2
+  skips:
+  - skupper-operator.v1.4.0-rc2
+  - skupper-operator.v1.4.0-rc3
```

Comment:
@fgiorgetti can you please submit a new pr to remove `replaces` and `skips`? Since this is the first bundle for this operator, it doesn't really do anything and it's causing issues in the index consistency check.

Reply:
@mporrato done through #3498.
Thank you.